test/core/OmnibugSettings.js

```diff
@@ -24,6 +24,7 @@ test("OmnibugSettings should return a list of defaults", t => {
     t.is(typeof settings.defaults.defaultPattern, "string");
     t.is(typeof settings.defaults.providers, "object");
     t.is(typeof settings.defaults.highlightKeys, "object");
+    t.is(typeof settings.defaults.renameParameters, "string");
```

Comment:
I'd say we force this into an object upon save; less manipulation later on is better.

Reply:
agreed, i meant to actually but forgot, i'll clean that up